config/routes.rb

```diff
@@ -1,58 +1,4 @@
 Rails.application.routes.draw do
   root 'homes#index'
   devise_for :users
-  # The priority is based upon order of creation: first created -> highest priority.
```

Comment:
There should be a route for your categories actions, for this it should have an only: [:index]. If you also want it to be your homepage you should be changing the root path to the index on the categories controller.